HIP/hip-583.md

```diff
@@ -10,220 +10,452 @@ status: Accepted
+permits the use of this `alias` for all operations supported by `alias` today. This HIP expands support in the
+auto-create flow for `ECDSA` derived addresses to be used as the `alias`. It also adds support to `CryptoCreate`
+transactions for creation with either a key-based `alias`, or an `ECDSA` derived `alias`.
+
+Rationale
+------------------------------------------------------------------------------------------------------------------------
+
+Fundamental in the design of the Hedera network are the concepts of _accounts_, account _identifiers_, and _keys_. These
+are fundamentally different entities. Today, a user `Account` must have a key. The key for the account may be a simple
+key (typically, an `ED25519` public key), or it may be a compound key, such as a list of keys all of which must sign
+each transaction, or a threshold key such that N of the keys must be signed. These may be nested to create very rich and
+powerful accounts requiring multiple-signatures. Further, an `Account` may change the keys on the account over time.
+
+An `Account` also has a system-assigned account identifier of the form `shard.realm.num`, where each value is an 8-byte
+number. An `Account` may also be referred to with an identifier of the form `shard.realm.alias`, where the alias is of
+one of the following forms:
+
+1. `account-num alias` - This is the Hedera account number styled as an Ethereum account hex string. The
+   Hedera account number is hex encoded and the value is prefixed with zeros to form a 20 byte address. This format
+   cannot be used to auto-create an account, but can be used wherever an `alias` can be used. This is sometimes
+   referred to as the "long-zero" alias, since the leftmost 12 bytes are 0.
+2. `key alias` - This format is created by using the bytes of a simple cryptographic public key supported by Hedera
+   (such as `ED25519` or `ECDSA secp256k1`). It is not necessary for the account's key to match the `alias`, only that
+   the account owner has asserted cryptographic proof of ownership of the account `alias` key in addition to the
+   account key. The key is protobuf-encoded and those raw bytes are included in the `alias`.
+3. `evm address alias` - This format is created by using the rightmost 20 bytes of the 32 byte `Keccak-256` hash of an
+   `ECDSA secp256k1` public key. This calculation is in the manner described by the Ethereum Yellow Paper.
+   Note, that the recovery id is not formally part of the public key and not included in the hash. This is
+   calculated on the consensus nodes using the `ECDSA` key provided in the auto create `CryptoTransfer` transaction.
+   While both `evm address alias` and `account-num alias` are 20 byte values, they can be differentiated by the fact
+   that `account-num alias` is always prefixed with 12 bytes. The likelihood of randomly generated ECDSA keys, being
+   hashed, having 12 sequential zeros is very low. Those keys, should they be found, cannot be used on the network.
+
```

Comment:
Add a 4th point that all accounts have `account-num alias` and _may_ have one additional alias that is either `key alias` or `evm address alias`.